test/utils.sh

```diff
@@ -227,3 +227,88 @@ get_image_manifests() {
   echo "${image_manifests}"
 
 }
+
+# This function will be used by tekton tasks in build-definitions
+# It returns a list of unreleased bundles in the FBC fragment by comparing it with
+# the corresponding production index image
+get_unreleased_bundle() {
+  # FBC fragment containing the unreleased bundle
+  local FBC_FRAGMENT=""
+
+  get_unreleased_bundle_usage()
+  {
+    echo "
+  get_unreleased_bundle  -i FBC_FRAGMENT
+  " >&2
+    exit 2
+  }
+
+  local OPTIND opt
+  while getopts "i:" opt; do
+      case "${opt}" in
+          i)
+              FBC_FRAGMENT="${OPTARG}" ;;
+          *)
+              get_unreleased_bundle_usage ;;
+      esac
+  done
+  shift $((OPTIND-1))
+
+  if [ -z "$FBC_FRAGMENT" ]; then
+    echo "Missing parameter FBC_FRAGMENT" >&2
+    exit 2
+  fi
+
+  local INDEX_IMAGE="registry.redhat.io/redhat/redhat-operator-index"
+
+  #Get target ocp version from the fragment
+  if ! ocp_version=$(skopeo inspect --no-tags --raw docker://"$FBC_FRAGMENT"); then
+    echo "The raw image inspect command failed" >&2
+    exit 1
+  fi
+  ocp_version=$(echo "$ocp_version" |  jq -r '.annotations."org.opencontainers.image.base.name"' | cut -d: -f2)
+
+  # Decide which OPM version to use for rendering the image
+  local opm_version
+  if [[ "$(printf '%s\n' "$ocp_version" "v4.17" | sort -V | head -n1)" == "v4.17" ]]; then
+    opm_version="opm-v1.40.0"
```

Comment:
If we are maintaining two versions, we would want to ensure that this is updated whenever the Dockerfile is updated.

Reply:
This is no longer applicable since we are using a single opm version